2021-09-07 04:58:31,438 INFO    MainThread:21024 [wandb_setup.py:_flush():69] setting env: {'entity': 'zumo09', 'project': 'flatland-rl-global-reward-sweep', 'sweep_id': 'z806yjo8', 'root_dir': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN', 'run_id': '14avl3ht', 'sweep_param_path': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN\\wandb\\sweep-z806yjo8\\config-14avl3ht.yaml'}
2021-09-07 04:58:31,438 INFO    MainThread:21024 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-07 04:58:31,438 INFO    MainThread:21024 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210907_045831-14avl3ht\logs\debug.log
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210907_045831-14avl3ht\logs\debug-internal.log
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_init.py:init():375] calling init triggers
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_init.py:init():382] wandb.init called with sweep_config: {'global_reward': 5, 'invalid_action_penalty': -1, 'step_penalty': -1.5}
config: {'n_episodes': 500, 'env_config': 0, 'n_evaluation_episodes': 5, 'checkpoint_interval': 100, 'eps_start': 1.0, 'eps_end': 0.01, 'eps_decay': 0.99, 'invalid_action_penalty': -1.0, 'step_penalty': -1.5, 'global_reward': 5.0, 'stop_penalty': 0.0, 'start_penalty': 0.0, 'buffer_size': 100000, 'buffer_min_size': 0, 'restore_replay_buffer': '', 'save_replay_buffer': False, 'batch_size': 128, 'gamma': 0.93, 'tau': 0.001, 'learning_rate': 8e-05, 'hidden_size': 128, 'update_every': 8, 'render': False, 'training_id': '210907045829', 'n_agents': 2, 'x_dim': 35, 'y_dim': 35, 'n_cities': 2, 'max_rails_between_cities': 2, 'max_rails_in_city': 3, 'malfunction_rate': 0.02, 'seed': 0, 'observation_tree_depth': 2, 'observation_radius': 10, 'observation_max_path_depth': 30}
2021-09-07 04:58:31,439 INFO    MainThread:21024 [wandb_init.py:init():424] starting backend
2021-09-07 04:58:31,440 INFO    MainThread:21024 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-09-07 04:58:31,446 INFO    MainThread:21024 [backend.py:ensure_launched():135] starting backend process...
2021-09-07 04:58:31,526 INFO    MainThread:21024 [backend.py:ensure_launched():140] started backend process with pid: 3976
2021-09-07 04:58:31,527 INFO    MainThread:21024 [wandb_init.py:init():429] backend started and connected
2021-09-07 04:58:31,529 INFO    MainThread:21024 [wandb_run.py:_config_callback():843] config_cb None None {'global_reward': 5, 'invalid_action_penalty': -1, 'step_penalty': -1.5}
2021-09-07 04:58:31,529 INFO    MainThread:21024 [wandb_init.py:init():477] updated telemetry
2021-09-07 04:58:31,530 INFO    MainThread:21024 [wandb_init.py:init():500] communicating current version
2021-09-07 04:58:33,276 INFO    MainThread:21024 [wandb_init.py:init():505] got version response 
2021-09-07 04:58:33,276 INFO    MainThread:21024 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-07 04:58:33,645 INFO    MainThread:21024 [wandb_init.py:init():540] starting run threads in backend
2021-09-07 04:58:35,989 INFO    MainThread:21024 [wandb_run.py:_console_start():1601] atexit reg
2021-09-07 04:58:35,991 INFO    MainThread:21024 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-07 04:58:35,992 INFO    MainThread:21024 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-07 04:58:35,994 INFO    MainThread:21024 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-07 04:58:35,994 INFO    MainThread:21024 [wandb_init.py:init():565] run started, returning control to user process
2021-09-07 05:59:51,353 INFO    MainThread:21024 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-07 05:59:51,353 INFO    MainThread:21024 [wandb_run.py:_restore():1543] restore
2021-09-07 05:59:51,926 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2021-09-07 05:59:52,062 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2021-09-07 05:59:52,306 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2021-09-07 05:59:52,555 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9549
}

2021-09-07 05:59:52,671 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9549
}

2021-09-07 05:59:52,779 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9549
}

2021-09-07 05:59:52,888 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:52,996 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:53,106 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:53,214 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:53,323 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:53,767 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:53,878 INFO    MainThread:21024 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9549
  total_bytes: 9549
}

2021-09-07 05:59:55,182 INFO    MainThread:21024 [wandb_run.py:_show_summary():1848] rendering summary
2021-09-07 05:59:55,186 INFO    MainThread:21024 [wandb_run.py:_show_history():1886] rendering history
2021-09-07 05:59:55,192 INFO    MainThread:21024 [wandb_run.py:_show_files():1915] logging synced files
